src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+    <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is ${person.sex === 'm'
+          ? 'my wife'
+          : 'my husband'}`
+        }
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
You can use the ternary operator inside a paragraph tag instead of re-rendering different paragraphs depending on the condition
```suggestion
    {person.isMarried ? (
          `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`
        ) : (
          'I am not married'
        )}
```